src/scripts/achievements/AchievementHandler.ts

```diff
@@ -20,7 +20,7 @@ class AchievementHandler {
     }
 
     public static calculateNumberOfTabs() {
-        this.numberOfTabs(Math.floor((this.achievementListFiltered().filter(a => a.region <= player.highestRegion()).length - 1) / 10));
+        this.numberOfTabs(Math.min(1, Math.floor(this.achievementListFiltered().filter(a => a.region <= player.highestRegion()).length / 10)));
```

Comment:
I think you may have reversed the logic here, as only 2 pages ever show up:
```suggestion
        this.numberOfTabs(Math.max(1, Math.floor(this.achievementListFiltered().filter(a => a.region <= player.highestRegion()).length / 10)));
```
![image](https://user-images.githubusercontent.com/7288322/95668839-77284700-0bd5-11eb-919c-79871a019d66.png)

Reply:
Oh goodness. I can't believe I did that xD Should be committed now